src/.nuget/packages.config

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
```

Comment:
Maybe add a comment here for what goes in the .xml list and what goes in the .json?  I don't understand :confused: